### PR TITLE
DPAD-1140 :: Integrates Ads through the AdEngine library

### PR DESCRIPTION
**Pull request by AlburIvan**
### Issue

[DPAD-1140](https://fandom.atlassian.net/browse/DPAD-1140)

### Description

The video-platform is not integrated with the AdEngine library. As a result of that, the video player on this new platform never plays ads. This is a major release blocker, given that the current `/video` page that’s served from the F2 repo does have ads.

### Links

https://build-dpad-1140.apps.fandom-dev.us/video - Ads enabled

### CODE REVIEW

File: src/jwplayer/players/CanonicalVideoPlayer/CanonicalVideoPlayer.tsx
Comment (p-forfa) [inline src/jwplayer/players/CanonicalVideoPlayer/CanonicalVideoPlayer.tsx:50]: Is "Dev" considered Prod in this case? It looks like in this case if its client-side, then this will resolve true for both a "prod" deployment and a "dev" deployments (whether its build-dev, dev or prod in the FAT ecosystem). Just wanted to make sure that this was the correct assumption.

File: src/jwplayer/players/CanonicalVideoPlayer/CanonicalVideoPlayer.tsx
Comment (p-forfa) [inline src/jwplayer/players/CanonicalVideoPlayer/CanonicalVideoPlayer.tsx:60]: 👍

File: src/jwplayer/players/CanonicalVideoPlayer/canonicalOnReady.ts
Comment (p-forfa) [inline src/jwplayer/players/CanonicalVideoPlayer/canonicalOnReady.ts:8]: Interesting. I didn't see this part in any of the other players from what I remember.

Comment (p-forfa) [reply src/jwplayer/players/CanonicalVideoPlayer/canonicalOnReady.ts:8]: Is this for tracking, or is this for AdEngine?

File: src/jwplayer/players/CanonicalVideoPlayer/CanonicalVideoPlayer.tsx
Comment (p-forfa) [reply src/jwplayer/players/CanonicalVideoPlayer/CanonicalVideoPlayer.tsx:50]: Ah, I just saw the isLocalDevelopment() definition. It looks like its the ENV variable that's set when a minified bundle is created, vs the un-minified bundle when you do local development.

**Review by p-forfa — APPROVED** [review]
LGTM. Looks like we did a merge with master sometime ago, so the branch shouldn't be stale. Once we go through a QA cycle and get approval, we can bump the package version and publish.

File: src/jwplayer/players/CanonicalVideoPlayer/canonicalOnReady.ts
Comment (AlburIvan) [reply src/jwplayer/players/CanonicalVideoPlayer/canonicalOnReady.ts:8]: Yup, it exists on [articlePlayerOnReady.ts](https://github.com/Wikia/jwplayer-fandom/blob/master/src/jwplayer/utils/articleVideo/articlePlayerOnReady.ts) for the UCP player too

**Pull request merged**
